app/src/main/java/io/bloco/circles/ui/auth/signup/ChooseUsernameActivity.kt

```diff
@@ -69,10 +79,10 @@ class ChooseUsernameActivity : BaseActivity() {
         viewModel
             .loading()
             .onEach { loading ->
-                skip.isVisible = !loading
                 continueBtn.isVisible = !loading
                 loadingSpinner.isVisible = loading
                 chooseUsernameLayout.isVisible = !loading
+                skip.isVisible = if(loading) false else canSkip
```

Comment:
Instead of having a local variable, you could flatMap `viewModel.canSkipUsername()`. But it's not a big deal.